hubstorage/job.py

```diff
@@ -20,15 +20,6 @@ def __init__(self, client, key, auth=None, jobauth=None, metadata=None):
         self.requests = Requests(client, self.key, self.auth)
         self.jobq = JobQ(client, self.key.split('/')[0], auth)
 
-    def close_writers(self):
```

Comment:
I think this method has a place in Job class. It allows scraper writers to correctly close the writers without a need to close all batchuploader writers.